drivers/scheduler/scheduler.go

```diff
@@ -220,6 +178,15 @@ type Driver interface {
 
 	// Get token for a volume
 	GetTokenFromConfigMap(string) (string, error)
+
+	// AddLabelOnNode adds key value labels on the node
+	AddLabelOnNode(node.Node, string, string) error
+
+	//IsAutopilotEnabledForPvc checks if autopilot enabled for a given PVC
+	IsAutopilotEnabledForPvc(map[string]string) bool
```

Comment:
API should be

```
IsAutopilotEnabledForVolume(v *volume.Volume)
```

2 issues with current method

1. It calls out PVC which is just a k8s concept
2. It says ForPvc but provides a map of strings. There is an implicit assumption is the API that volumes have annotations or labels which is a map[string]string